Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04 as build
+ENV TZ=UTC
+
+LABEL Cammoguy <jpburnett@gwu.edu>
```

Comment:
What is `LABEL` intended for? Can it have a bit of a more generic name?

Reply:
I think it is just fluff. I was using other files as reference and used as the "maintainer" so I put it in there so people knew who to yell at if it wasn't working 😅